ingestion/src/airflow_provider_openmetadata/lineage/openmetadata.py

```diff
@@ -41,22 +42,36 @@ class OpenMetadataLineageConfig(ConfigModel):
 
 def get_lineage_config() -> OpenMetadataLineageConfig:
     """Load the lineage config from airflow_provider_openmetadata.cfg."""
-    airflow_service_name = conf.get(
-        "lineage", "airflow_service_name", fallback="airflow"
-    )
-    api_endpoint = conf.get(
-        "lineage", "openmetadata_api_endpoint", fallback="http://localhost:8585"
-    )
-    auth_provider_type = conf.get("lineage", "auth_provider_type", fallback="no-auth")
-    secret_key = conf.get("lineage", "secret_key", fallback=None)
-    return OpenMetadataLineageConfig.parse_obj(
-        {
-            "airflow_service_name": airflow_service_name,
-            "api_endpoint": api_endpoint,
-            "auth_provider_type": auth_provider_type,
-            "secret_key": secret_key,
-        }
-    )
+    airflow_service_name = conf.get("lineage", "airflow_service_name", fallback=None)
+    if airflow_service_name:
+        api_endpoint = conf.get(
+            "lineage", "openmetadata_api_endpoint", fallback="http://localhost:8585"
+        )
+        auth_provider_type = conf.get(
+            "lineage", "auth_provider_type", fallback="no-auth"
+        )
+        secret_key = conf.get("lineage", "secret_key", fallback=None)
+        return OpenMetadataLineageConfig.parse_obj(
+            {
+                "airflow_service_name": airflow_service_name,
+                "api_endpoint": api_endpoint,
+                "auth_provider_type": auth_provider_type,
+                "secret_key": secret_key,
+            }
+        )
+    else:
```

Comment:
The else's in this method can be removed to reduce indentation, since the previous if's return from the method.

```python
if airflow_service_name:
  ... 
  return OpenMetadataLineageConfig.parse_obj(...)

openmetadata_config_file = os.getenv("OPENMETADATA_LINEAGE_CONFIG")
if openmetadata_config_file:
  ...
  return OpenMetadataLineageConfig.parse_obj(config)

return OpenMetadataLineageConfig.parse_obj(...)

```